### PR TITLE
Fix bug + other stuff

### PR DESCRIPTION
**Pull request by RDIL**
## Description  
  
This fixes a bug in victory royale mod that causes it to play the sound even when it is not on.  This also does some quick cleanup.  

## Context  
  
I personally experienced this and have tested it after the fix.  I can confirm this prevents it from happening.  

## Anything Else  

Tested and working

## Checklist  

- [X] All *new* Java and Kotlin files have the right copyright header  
- [X] I have tested my code by building it locally  
- [X] This is not a work-in-progress and is ready for merge  
- [X] I am submitting the PR under and understand the terms of the GNU Lesser General Public License v3.0

### CODE REVIEW

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java [hunk 1]
```diff
@@ -18,20 +18,21 @@
...
 public class VictoryRoyale extends AbstractMod {
-
     private static VictoryRoyale INSTANCE;
 
     private ResourceLocation texture = new ResourceLocation("textures/material/victoryroyale/victory_royale.png");
     private ConcurrentLinkedQueue<WhiteLine> points = new ConcurrentLinkedQueue<>();
 
     private long start = 0;
 
+    @Contract(pure = true)
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: Why is this necessary?

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java [hunk 2]
```diff
@@ -160,14 +152,17 @@ public void onRender(RenderHUDEvent event) {
 
             GlStateManager.color(1, 1, 1, in < 1000 ? (float) Math.pow(in / 1000D, 2) : fadeOutAlpha);
             GlStateManager.translate(imageWidth, 10, 0);
+
             Gui.drawScaledCustomSizeModalRect(0, 0, 0, 0, 1200, 675, imageWidth * 2, imageWidth, 1200, 675);
+            System.out.println("victory royale (debug)");
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: You've left a debug statement in the code.

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java [hunk 3]
```diff
@@ -200,11 +199,10 @@ public Metadata getModMetadata() {
     }
 
     class WhiteLine {
-
         private double xPercent, yPercent;
         private int width;
 
-        public WhiteLine(double xPercent, double yPercent, int width) {
+        protected WhiteLine(double xPercent, double yPercent, int width) {
```
Comment (CoalCoding) [inline src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: Why is this necessary?

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java (hunk 1 above)
Comment (RDIL) [reply src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: Sorry, IntelliJ has a mind of its own.  Removed.

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java (hunk 2 above)
Comment (RDIL) [reply src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: Outdated branch 🙁 
Removed.

File: src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java (hunk 3 above)
Comment (RDIL) [reply src/main/java/cc/hyperium/mods/victoryroyale/VictoryRoyale.java]: Sorry, IntelliJ has a mind of its own. Removed.
